tests/test_logger.py

```diff
@@ -0,0 +1,30 @@
+import json
+from datetime import datetime as real_datetime
+from unittest import TestCase
+from unittest.mock import MagicMock, patch
+
+from supertokens_python.constants import VERSION
+from supertokens_python.logger import (log_debug_message, streamFormatter,
+                                       streamHandler)
+
+
+class LoggerTests(TestCase):
+    @patch('supertokens_python.logger.datetime', wraps=real_datetime)
+    def test_json_msg_format(self, datetime_mock: MagicMock):
+        datetime_mock.utcnow.return_value = real_datetime(2000, 1, 1)  # type: ignore
+
+        with self.assertLogs(level='DEBUG') as captured:
+            log_debug_message("API replied with status 200")
+
+        record = captured.records[0]
+        streamHandler.transform(record)
+        out = json.loads(record.msg)
+        filename, lineno = out.pop('file').split(':')
+
+        assert int(lineno) > 0
+        assert filename.endswith('tests/test_logger.py')
```

Comment:
Explain why we are popping the file attribute in the JSON and checking that separately.

Reply:
Previously it was showing absolute path. 

Using relative path now. No need to pop. Updated the code.